wiremock/deployment/src/main/java/io/quarkiverse/openapi/wiremock/generator/deployment/wiremock/model/Stubbing.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkiverse.openapi.wiremock.generator.deployment.wiremock.model;
+
+import java.util.Objects;
+
+/**
+ * Represents a basic Wiremock stub.
+ * See <a href="https://wiremock.org/docs/stubbing/#basic-stubbing">here</a>
+ * the official Wiremock documentation for more details.
+ */
+public record Stubbing(Request request, Response response) {
+
+    public Stubbing(Request request, Response response) {
+        this.request = Objects.requireNonNull(request);
+        this.response = Objects.requireNonNull(response);
+    }
+
+    public String urlPath() {
+        return this.request.getUrlPathTemplate();
+    }
+
+    public String method() {
+        return this.request.getMethod();
+    }
+
+    public static class StubbingBuilder {
```

Comment:
Do we really need a builder?

Reply:
Actually no, we can add it when we have more attribute here.